scripts/sommelier

```diff
@@ -298,6 +308,7 @@ append_dir PATH "$WINE_PLATFORM/bin"
 # Note: We don't need to manually specify WINEDLLPATH, `wine` can find them on
 #       its own.
 export WINELOADER="$WINE_PLATFORM/bin/wine"
+export MSIEXEC="$WINE_PLATFORM/bin/msiexec"
```

Comment:
You can remove this line when you use `"${WINELOADER}" msiexec /i ...` as stated above.

Reply:
Done.